client/allocrunner/taskrunner/driver_handle.go

```diff
@@ -80,6 +83,9 @@ func (h *DriverHandle) ExecStreaming(ctx context.Context,
 	command []string,
 	tty bool,
 	stream drivers.ExecTaskStream) error {
+	if h == nil {
+		return ErrTaskNotRunning
+	}
```

Comment:
These changes aren't really necessary now, but they'll provide belt-and-suspenders against future bugs.